ci/run.sh

```diff
@@ -43,8 +43,8 @@ rm -f $rlib_paths
 
 cargo build --target "$target"
 cargo build --target "$target" --release
-cargo build --target "$target" --features c
-cargo build --target "$target" --release --features c
+# cargo build --target "$target" --features c
+# cargo build --target "$target" --release --features c
```

Comment:
`--features c` requires the LLVM compiler-rt library in C, but we don't ship that with platform tools:

https://github.com/rust-lang/compiler-builtins/blob/3dea633a80d32da75e923a940d16ce98cce74822/build.rs#L578-L585